.github/workflows/release.yaml

```diff
@@ -10,8 +10,8 @@ jobs:
 
       - uses: actions/cache@v2
         with:
-          path: .cache
-          key: nix-v1-${{ hashFiles('nix/nixpkgs.json') }}
+          path: /nix
```

Comment:
@giuseppe I am now trying to cache /nix directly, rather than placing the cache under ./.cache